### DISCUSSION TITLE
0844. 比较含退格的字符串

### DISCUSSION THREAD

**Discussion by giscus** [itcharge/AlgoNote#88] [Comments]
# 0844. 比较含退格的字符串 | 算法通关手册

[https://algo.itcharge.cn/solutions/0800-0899/backspace-string-compare/](https://algo.itcharge.cn/solutions/0800-0899/backspace-string-compare/)

**Comment by Is-Grit** [comment]
这个题感觉用二分法好麻烦，堆栈好评！
